resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -30,7 +30,7 @@
                                 <tr>
                                     <td><center><a data-toggle="tooltip" title="View Details" href="/dashboard/controllers/profile/feedback-details/{{ $f->id }}">{{ $f->position }}</a></center></td>
                                     <td><center>{{ $f->service_level_text }}</center></td>
-                                    <td><center>{{ str_limit($f->comments, 50, '...') }}</center></td>
```

Comment:
What is this (seemingly random) change? Seems unrelated to the issue, although I might be missing context

Reply:
It's an unrelated change, but since I was in the dashboard fixing stuff I decided to correct this. Fixes a formatting error in the dashboard that was causing columns to grow too large.

Reply:
Got it.